source/guides/clear/mixer.rst

```diff
@@ -411,27 +479,197 @@ mix version 10 (from which the image was built), to mix version 20.
       swupd update
       swupd bundle-list -a
 
-#. Now your mix should be at version 20 and curl is now available. Try using
+#. Now your mix should be at version 20 and curl is available. Try using
    curl. This will fail because curl is not yet installed:
 
    .. code-block:: console
 
       curl: command not found
       To install curl use: swupd bundle-add curl
 
-   Add the new bundle from your update server to your VM. Retry curl. It works!
+#. Add the new bundle from your update server to your VM. Retry curl.
+   It works!
 
    .. code-block:: bash
 
       swupd bundle-add curl
       curl -O https://download.clearlinux.org/image/start_qemu.sh
 
-   Shutdown your VM:
+#. Shutdown your VM:
 
    .. code-block:: bash
 
       poweroff
 
+Example 6: Remove a bundle
```

Comment:
I think the title of Example 6 should be "Remove a bundle from client system in future release"